server/src/main/java/org/elasticsearch/action/admin/indices/shards/IndicesShardStoresResponse.java

```diff
@@ -232,29 +233,33 @@ public Failure(String nodeId, String index, int shardId, Throwable reason) {
             this.nodeId = nodeId;
         }
 
-        private Failure() {
+        private Failure(StreamInput in) throws IOException {
+            if (in.getVersion().before(Version.V_7_4_0)) {
+                nodeId = in.readString();
+            }
+            readFrom(in, this);
+            if (in.getVersion().onOrAfter(Version.V_7_4_0)) {
+                nodeId = in.readString();
+            }
         }
 
         public String nodeId() {
             return nodeId;
         }
 
         static Failure readFailure(StreamInput in) throws IOException {
```

Comment:
I missed this, but this seems unnecessary too

Reply:
Given we have these all over, and haven't removed them in other PRs, I'm going to leave these for now.